content/blog/2023-12-11-btor2i.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "An Interpreter for the btor2 Model Checking Language"
+[extra]
+bio = """
+  Sanjit, Omkar, and Justin are all MEng students in CS at Cornell.
+"""
+[[extra.authors]]
+name = "Sanjit Basker"
+link = "https://github.com/SanjitBasker"
+[[extra.authors]]
+name = "Omkar Bhalerao"
+link = "https://github.com/obhalerao" 
+[[extra.authors]]
+name = "Justin Ngai"
+link = "https://github.com/NgaiJustin"
++++
+
+[Link to project repository](https://github.com/NgaiJustin/btor2i)
```

Comment:
What do you think about working this in more elegantly below? You could link the first time you mention that you have implemented an interpreter, for example. :smiley: